src/amo/pages/UserProfileEdit/index.js

```diff
@@ -465,14 +465,16 @@ export class UserProfileEditBase extends React.Component<Props, State> {
 
     if (!currentUser) {
       return (
-        <div className="UserProfileEdit">
-          <Card className="UserProfileEdit-authenticate">
-            <AuthenticateButton
-              noIcon
-              logInText={i18n.gettext('Log in to edit the profile')}
-            />
-          </Card>
-        </div>
+        <Page>
```

Comment:
It would be nicer to move this to a variable like `output` so that `render()` can wrap *any* kind of output in `<Page>`. This approach might prevent the bug from happening again.

This is just a suggestion for an enhancement and doesn't need to block landing this feature.

Reply:
Good idea. I'll make that change.